lms/envs/common.py

```diff
@@ -924,6 +924,10 @@
 
 # Localization strings (e.g. django.po) are under this directory
 LOCALE_PATHS = (REPO_ROOT + '/conf/locale',)  # edx-platform/conf/locale/
+
+# TODO: Figure out a way to discover locale paths for installed components
+LOCALE_PATHS = ('/edx/src/drag-and-drop-v2/conf/locale',) + LOCALE_PATHS
```

Comment:
@asadiqbal08 and I discussed this late last night and we're also thinking it's not needed.